UIDD/UIDD-Session7/form.html

```diff
@@ -0,0 +1,122 @@
+                        <input type="text" id="last_name" name="last_name" placeholder="Last Name" title="Enter your last name">
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="label">
+                        <label>E-Mail</label>
+                    </div>
+                    <div class="icon">
+                        <i class="fas fa-envelope"></i>
+                    </div>
+                    <div class="content">
+                        <input type="text" id="email" name="email" placeholder="E-Mail Address" title="Enter your email">
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="label">
+                        <label>Phone #</label>
+                    </div>
+                    <div class="icon">
+                        <i class="fas fa-phone"></i>
+                    </div>
+                    <div class="content">
+                        <input type="text" id="phone" name="phone" placeholder="(845)555-1212" title="Enter your phone">
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="label">
+                        <label>Address</label>
+                    </div>
+                    <div class="icon">
+                        <i class="fas fa-home"></i>
+                    </div>
+                    <div class="content">
+                        <input type="text" id="address" name="address" placeholder="Address" title="Enter your Address">
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="label">
+                        <label>City</label>
+                    </div>
+                    <div class="icon">
+                        <i class="fas fa-home"></i>
+                    </div>
+                    <div class="content">
+                        <input type="text" id="city" name="city" placeholder="City" title="Enter your City">
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="label">
+                        <label>State</label>
+                    </div>
+                    <div class="icon">
+                        <i class="fas fa-list-ul"></i>
+                    </div>
+                    <div class="content">
+                        <select id="state" name="state">
+                            <option value="">Please select your state</option>
+                            <option value="Rajasthan">Rajasthan</option>
+                            <option value="Haryana">Haryana</option>
+                            <option value="Maharashtra">Maharashtra</option>
+                        </select>
+                    </div>
+                </div>
+
+                <div id="submit_container" class="row">
+                    <div id="btn">
```

Comment:
why do we have id and name with "btn". could have pick something else regarding this.